Doc/library/tarfile.rst

```diff
@@ -229,7 +229,11 @@ details.
 
 .. data:: DEFAULT_FORMAT
 
-   The default format for creating archives. This is currently :const:`GNU_FORMAT`.
+   The default format for creating archives. This is currently :const:`PAX_FORMAT`.
+
+
+.. versionchanged:: 3.8
```

Comment:
I think this needs additional indentation. "versionchanged" is corresponded to "DEFAULT_FORMAT", not to the whole module.

Reply:
Great catch, thanks. Done.